frontend/routes/Wifi.jsx

```diff
@@ -12,42 +29,235 @@ export class Component extends React.Component {
       wifi_networks: [],
       last_refresh: '',
     };
+
     setInterval(() => {
-      exec_table('nmcli dev wifi').then(result => this.setState({ wifi_networks: result }));
-      exec('date').then(result => this.setState({ last_refresh: result }));
+      exec_table('nmcli dev wifi').then(result => {
+        if (!result) return;
+
+        this.setState({
+          wifi_networks: result,
+        });
+      });
+      exec('date').then(result => {
+        this.setState({
+          last_refresh: result,
+        });
+      });
     }, 1000);
   }
 
   render() {
     return (
       <div>
-        <Table data={this.state.wifi_networks} />
+        <EnhancedTable rows={this.state.wifi_networks} />
       </div>
     );
   }
 }
 
-function Table(props) {
-  const { data } = props;
-  if (data.length === 0) return <table />;
+
+// Meta-Data
```

Comment:
Unnecessary comment (also the one below).

Reply:
Removed